critiquebrainz/db/review.py

```diff
@@ -29,7 +29,23 @@
     "artist",
     "label",
     "recording",
-    "bb_edition_group"
+    "bb_edition_group",
+    "bb_literary_work"
+]
+
+MUSICBRAINZ_ENTITY_TYPES = [
+    "event",
+    "place",
+    "release_group",
+    "work",
+    "artist",
+    "label",
+    "recording"
+]
+
+BOOKBRAINZ_ENTITY_TYPES = [
+    "bb_edition_group",
+    "bb_literary_work"
```

Comment:
can we set `ENTITY_TYPES = MUSICBRAINZ_ENTITY_TYPES + BOOKBRAINZ_ENTITY_TYPES` here? (I thought I saw some other code where you did this, but I can no longer find it)